### PR TITLE
Update the power query template to include only one worksheet

### PR DESCRIPTION
**Pull request by mabezen**
Update the power query template to include only one worksheet - named sheet1

### CODE REVIEW

File: tests/workbookQueryTemplate.test.ts [hunk 1]
```diff
@@ -40,7 +40,7 @@ describe("Single query template tests", () => {
         expect(path).toEqual(item1Path);
     });
 
-    test("ConnectedWorkbook XML exists as item2.xml", async () => {
+    test("ConnectedWorkbook XML exists as item1.xml", async () => {
```
Comment (shanialbeck) [inline tests/workbookQueryTemplate.test.ts:43]: Why is this moved from items2 to items1? The new template saves connected-workbooks in items1?

**Review by shanialbeck — APPROVED** [review]

File: tests/workbookQueryTemplate.test.ts (hunk 1 above)
Comment (mabezen) [reply tests/workbookQueryTemplate.test.ts:43]: Yes, it was replaced after save
